### ISSUE TITLE
Build error when `env-filter` feature is not specified

### ISSUE THREAD

**Issue by d-e-s-o** [tokio-rs/tracing#518]
### Version

`tracing-subscriber-0.1.6`

### Platform

Linux 4.19, 64 bit

### Description

I am getting the following build error:
```
error[E0433]: failed to resolve: maybe a missing crate `EnvFilter`?
   --> tracing-subscriber-0.1.6/src/fmt/mod.rs:827:33
    |
827 |         .with_env_filter(crate::EnvFilter::from_default_env())
    |                                 ^^^^^^^^^ maybe a missing crate `EnvFilter`?
```

That happens with:
```toml
tracing-subscriber = {version = "0.1.6", default-features = false, features = ["chrono", "fmt"]}
```
I have to add the `env-filter` feature to make it go away.

**Comment by hawkw** [comment]
Thanks for the report! I *believe* this issue no longer exists in the `tracing-subscriber` 0.2.x series, which is currently in alpha, so when 0.2.0 is stable, this should be fixed.

**Comment by hawkw** [comment]
@d-e-s-o would you mind checking if this still happens on `tracing-subscriber` from the current Git master? I'm planning on publishing `tracing-subscriber` 0.2.0-alpha.3 today, and I'd like to make sure this is fixed before publishing.

**Comment by d-e-s-o** [comment]
Yes, still seeing it there.
```
error[E0433]: failed to resolve: maybe a missing crate `EnvFilter`?
   --> tracing-subscriber-0.2.0-alpha.2/src/fmt/mod.rs:661:33
    |
661 |         .with_env_filter(crate::EnvFilter::from_default_env())
    |                                 ^^^^^^^^^ maybe a missing crate `EnvFilter`?
```

**Comment by d-e-s-o** [comment]
Uhm, sorry. I tested with upstream `0.2.0-alpha.2`. Checked against `master` now and compilation seems to get past this point, so I think it is fixed there indeed.

**Comment by d-e-s-o** [comment]
I gather you don't want to fix it on the 0.1 branch. If so, feel free to close this issue.

**Comment by hawkw** [comment]
> I gather you don't want to fix it on the 0.1 branch. If so, feel free to close this issue.

We can backport fixes if there's user demand for it. We've backported changes to previous releases in the past & I'm willing to do so again. I mostly wanted to ensure that this issue _was_ fixed on 0.2.x.

**Comment by d-e-s-o** [comment]
Would be great if we could have it in 0.1.x, unless 0.2 stabilizes, say, tomorrow :-)

**Comment by hawkw** [comment]
Well, I'm hoping 0.2.0-alpha.3 will be the last alpha before 0.2.0 stable. But, I'm happy to fix it on 0.1.x as well if that's helpful!

**Comment by hawkw** [comment]
Closing this as the issue was resolved in 0.2.0, and 0.2.0 is now stable.
